pkg/test/helm/helm.go

```diff
@@ -38,33 +38,6 @@ func Init(homeDir string, clientOnly bool) error {
 	return err
 }
 
-// RepoAdd calls "helm repo add"
```

Comment:
Same here. If we can keep these around that will be great.  We've gone through so much churn with the installers that we keep writing similar utilities over and over. I'd rather keep some of these around for the next cycle.

Reply:
Done